lectures/markov_asset.md

```diff
@@ -19,26 +20,28 @@ kernelspec:
 In this lecture we consider some asset pricing problems and use them to
 illustrate some foundations of JAX programming.
 
-Most of the heavy lifting will be done through routines from linear algebra.
+Most of the heavy lifting is done through routines from linear algebra.
 
 Along the way, we will show how to solve some memory-intensive problems with large state spaces.
 
 We do this using elegant techniques made available by JAX, involving the use of linear operators to avoid instantiating large matrices.
 
-If you wish to skip all motivation and move straight to the first equation we plan to
-solve, you can jump to [](eq:ntecx2).
+If you wish to skip all motivation and move straight to the first equation we plan to solve, 
+you can jump to [](eq:ntecx2).
 
 In addition to what's in Anaconda, this lecture will need the following libraries:
 
-```{code-cell} ipython3
+```{code-cell}
```

Comment:
Thanks @chappiewuzefan. This change should be reverted.

Reply:
Sorry Smit, I'm confused here,

```{code-cell}
:tags: [hide-output]

!pip install quantecon
```

or I need to change to without hide-output?